lib/chef-dk/policyfile/storage_config.rb

```diff
@@ -38,6 +38,9 @@ def initialize(options = {})
       end
 
       def use_policyfile(policyfile_filename)
+        if policyfile_filename.end_with?(".lock.json")
+          policyfile_filename = policyfile_filename.sub(/\.lock.json\Z/, '.rb')
```

Comment:
maybe a little cleaner to do `return use_policyfile_lock(policyfile_filename)` here